InomialClient.js

```diff
@@ -8,8 +8,8 @@
 //
 
 module.exports = {
-    connect: function(hostname, stage, origin, apikey) {
-        const client = new InomialClient(hostname, stage, origin, apikey);
+    connect: function(hostname, stage, origin, apikey, websocketClientConfig) {
```

Comment:
is it possible to make this a property that you set rather than a field in the constructor? the majority of users will never need this, so it should be "possible" but not get in the way of "easy".

perhaps a static function such as `InomialClient.setWebsocketClientConfig`.

Reply:
I've just made that change now.